test/Helper/HeadScriptTest.php

```diff
@@ -534,4 +534,18 @@ public function testSupportsAsyncAttribute()
         $test = $this->helper->__invoke()->toString();
         $this->assertContains('async="', $test);
     }
+
+    /**
+     * @group 23
+     */
+    public function testOmitsTypeAttributeIfNoneGivenAndHtml5Doctype()
+    {
+        $view = new \Laminas\View\Renderer\PhpRenderer();
+        $view->plugin('doctype')->setDoctype(\Laminas\View\Helper\Doctype::HTML5);
+        $this->helper->setView($view);
+
+        $this->helper->__invoke()->appendScript('// some script' . PHP_EOL);
+        $test = $this->helper->__invoke()->toString();
+        $this->assertNotContains('type', $test);
```

Comment:
Expand this to `'type="text/javascript"`, to make it absolutely clear what we're trying NOT to emit.